src/pages/workspace/distanceRates/UnitSelector/index.tsx

```diff
@@ -41,7 +41,7 @@ function UnitSelector({defaultValue, wrapperStyle, label, setNewUnit}: UnitSelec
         hidePickerModal();
     };
 
-    const title = Str.recapitalize(translate(getUnitTranslationKey(defaultValue)));
+    const title = defaultValue ? Str.recapitalize(translate(getUnitTranslationKey(defaultValue))) : '';
```

Comment:
If `defaultValue` can be `undefined`, then we need to mark it as optional in `UnitSelectorProps`

Reply:
I agree, lets make this field optional

Reply:
@nkdengineer ^

Reply:
@MonilBhavsar @srikarparsi  I updated.